modular-docs-manual/content/topics/module_guidelines-assembly.adoc

```diff
@@ -27,7 +27,7 @@ Use the second level heading syntax for the Prerequisites section in the assembl
 [discrete]
 == Assembly Modules
 
-List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly.
+List link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include files] to include the required modules. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. You can use a `leveloffset` attribute to set the hierarchy of the module relative to the assembly. In most cases, create modules whose title is an H1 and include those modules by using the format `module-file.adoc[leveloffset=+1]`. Otherwise, if necessary, you can use other `leveloffset` values, such as `module-file.adoc[leveloffset=+2]`, to modify the structure of your module content.
```

Comment:
personally I think the mention of H1 here is confusing..  ie it's a separate issue and mentioning it here makes the text more ambiguous.

Reply:
> personally I think the mention of H1 here is confusing.. ie it's a separate issue and mentioning it here makes the text more ambiguous.

I don't think it is separate. It is integral to the notion of using leveloffset attributes. I'll see how I can clear up the ambiguity.

Reply:
Revising now. Please hold off on more comments until I'm done.

Reply:
Actually, I think Stetson's current revision encompasses all our concerns:
- It discourages "bad behavior" such as H-whatever + leveloffset=whatever
- It allows using leveloffset values to demote/promote module content.

Side note: I think we should avoid the term "nesting" in this discussion because that describes including something **within** something else, like [3.2.5. Nesting Assemblies in Assemblies](https://redhat-documentation.github.io/modular-docs/#nesting-assemblies). In that sense, we're not really talking about nesting modules within modules here. Here we're talking about using leveloffset to demote/promote modules relative to other modules in an assembly. Nesting modules within modules is a controversy we don't want to get into.

Reply:
I agree with @pwright . Every module and assembly title must be an H1 so  no need to mention it here. @rkratky you cannot use H2s in procedure modules and the only H1 is the title. You can use H2s in concept and reference modules.

Reply:
Hey folks. I'm sorry, but I find that updating confusing. Please consider this, which I think gets to the heart of the problem:

To include modules in an assembly, use the Asciidoc  link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#include-files[include directive]. Use any combination of concept, procedure, and reference modules that fulfills the purpose of the assembly. Use the`leveloffset` attribute to set the hierarchy of the module relative to the assembly, for example:
[source]
----
file1.adoc[leveloffset=+1]
file2.adoc[leveloffset=+2]
file3.adoc[leveloffset=+3]
----

NOTE: All module and assembly titles must use the H1 heading designator.

Reply:
I have no idea why the text is bolded, but you get the idea.

Reply:
Agreed with @emmurphy1 and @pwright on this. So @rolfedh and @rkratky , how would you feel about not getting into H1 to avoid the distraction from the point here?

Reply:
thanks @emmurphy1 I prefer your wording

Reply:
+1 @rolfedh @rkratky You good here?

Reply:
@sterobin All good. I trust your judgement!

Reply:
@emmurphy1 Done